.github/workflows/release.yml

```diff
@@ -77,11 +79,8 @@ jobs:
         run: chmod +x ./${{ env.EXECUTABLE }}
       - name: Set Tag Version
```

Comment:
```suggestion
      - id: get-version
        name: Get the con-pca-tasks Version
```

Reply:
@mcdonnnj resolved with [d8c5f32](https://github.com/cisagov/con-pca-tasks/pull/8/commits/d8c5f32d65777e1acfd9527a1637fd927213cde0)

Reply:
@itsmostafa Please update the step name as in the suggestion.

Reply:
@mcdonnnj resolved with [bbd95f5](https://github.com/cisagov/con-pca-tasks/pull/8/commits/bbd95f5dcd5135395c9e3952c08c4ad086f6e3dd)